public/app/features/auth-config/AuthConfigPage.tsx

```diff
@@ -63,7 +64,13 @@ export const AuthConfigPageUnconnected = ({ providerStatuses, isLoading, loadSet
       documentation.
     </a>
   );
+
   const subTitle = <span>Manage your auth settings and configure single sign-on. Find out more in our {docsLink}</span>;
+
+  const onCTAClick = () => {
+    reportInteraction('authentication_ui_created', { provider: firstAvailableProvider?.type });
```

Comment:
curious what firstAvailableProvider is?

Reply:
aha so we save the provider that started makes sense. I guess we need to go over more metrics but for now this is a good start